### DISCUSSION TITLE
Ballpark value of vector_utilization metric to consider vector is choked

### DISCUSSION THREAD

**Discussion by ShahroZafar** [vectordotdev/vector#20808] [Q&A] [answered]
Vector have a metric named `vector_utilization`. We have a system which is very critical for which we are reading logs and cannot lose logs. 
We want to get alerted when vector is lagging behind / fully choked and is applying backpressure. At what value of this metric should we consider that vector is working slowly and something needs to be done

**Comment by jszwedko** [comment, marked as answer]
A value close to `1` indicates that a component is a likely bottleneck since that indicates that the component is never waiting for input.
